File: src/main/java/app/lunchtime/core/controller/VersionController.java
package app.lunchtime.core.controller;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class VersionController {

    @Value("${application.version}")
    private String appVersion;

    @RequestMapping
    public String appVersion() {
        return appVersion;
    }
}

File: src/main/kotlin/app/lunchtime/core/controller/VersionController.kt
package app.lunchtime.core.controller

import org.springframework.beans.factory.annotation.Value
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class VersionController {

    @Value("\${application.version}")
    lateinit var appVersion: String

    @RequestMapping
    fun appVersion(): String {
        return appVersion
    }
}
